### PR TITLE
Add Local Compression Option

### PR DESCRIPTION
**Pull request by SoumyBhow**
This PR aims to install CHDMAN and Maxcso with the Pegasus pack along with scripts to allow easy local compression. Maxcso is used for PPSSPP and CHDMAN is used for other platforms like Saturn, PSX, PS2, etc. I redid everything from scratch and it should work now. Only mild issue is that I can't get the compressor script to output in stdout and the log file. I tried ```tee``` but that broke the script.

### CODE REVIEW

**Comment by dollerbill** [comment]
@SoumyBhow thank you for reopening this PR! 
Everything looks great in the quick look I did, hopefully I'll be able to test it this weekend and we can look at getting it merged

**Comment by SoumyBhow** [comment]
> 

Glad you liked it! Thing I'm most worried about rn is the fact that (for now) it involves ```cd```ing into each folder and compressing there. Not sure how that'll fair with the rest of the script but I tested it on Android 12L and I had no issues with that..

File: startup.sh [hunk 1]
```diff
@@ -100,13 +107,7 @@ if [[ $selected_option == "" ]]
...
+if [[ $selected_option == "9" ]]
 then
 	/bin/bash ~/dragoonDoriseTools/termux_pkg_install.sh
 fi
-
-
-if [[ $selected_option == "9" ]]
-then
-	am startservice -a com.termux.service_stop com.termux/.app.TermuxService &> /dev/null
-fi
```
Comment (dollerbill) [inline startup.sh:112]:
```
if [[ $selected_option == "10" ]]
then
	am startservice -a com.termux.service_stop com.termux/.app.TermuxService &> /dev/null
fi 
```

Let's get the exit option added back in

File: compress.sh [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	useInternalStorage=true
+	storageLocation="shared/roms"
+else
+	useInternalStorage=false
+	storageLocation="external-1"
+
+fi
+
+#Pick the systems
+SYSTEMS=$(whiptail --title "ROM Compressor" --checklist "Please select the systems whose ROMs you want to compress" 20 78 15 \
```
Comment (dollerbill) [inline compress.sh]:
```suggestion
SYSTEMS=$(whiptail --title "ROM Compressor" --checklist "Please select the systems whose ROMs you want to compress" 20 78 5 \
```

To cleanup a bit of the extra highlighting below the options

File: compress.sh [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    compressMaxcso &>> ~/storage/shared/pegasus_installer_log.log
+    else
+    compressCHDMAN &>> ~/storage/shared/pegasus_installer_log.log
+    fi
+	if [ "$removeOldROM" = "YES" ]; then
+    rm -rf ./*.{iso,cue,bin} &>> ~/storage/shared/pegasus_installer_log.log
+    fi
+done
+clear
+ echo -e "You may need to re-scrap your games after they've been compressed!"
```
Comment (dollerbill) [inline compress.sh:65]: If we wanted to we could probably check the metadata file for an existing iso/etc and replace that line with the newly scraped game path.. but it wouldn't hurt to leave as-is and add that in later

**Review by dollerbill — CHANGES_REQUESTED** [review]
Looks good, just a few really small things! I'm still testing the actual compression but hopefully I can wrap that up soon

**Comment by SoumyBhow** [comment]
@dollerbill Requested changes have been made. I also changed the compressor script so that instead of converting all of the ROMs and deleting the uncompressed versions in one go, it converts a ROM then deletes the old ROM one by one. This makes the script safer and doesn't eat up storage as it previously did.

**Comment by dollerbill** [comment]
> @dollerbill Requested changes have been made. I also changed the compressor script so that instead of converting all of the ROMs and deleting the uncompressed versions in one go, it converts a ROM then deletes the old ROM one by one. This makes the script safer and doesn't eat up storage as it previously did.

this is a great addition, one of my earlier concerns was potentially eating up all available storage before finishing the conversion. Glad you added it!

Taking another look now

**Review by dollerbill — APPROVED** [review]
Looks great! I'm gonna keep testing a bit on my device and will get it merged soon

**Comment by SoumyBhow** [comment]
Thank you so much. Glad it's looking good.

**Pull request merged**